libs/src/AudiusLibs.ts

```diff
@@ -536,6 +540,11 @@ export class AudiusLibs {
         ...this.discoveryProviderConfig
       })
       await this.discoveryProvider.init()
+
+      // set discovery provider in web3 for relay
+      if (this.web3Config && this.useDiscoveryRelay) {
+        this.web3Manager?.setDiscoveryProvider(this.discoveryProvider)
```

Comment:
this could silently not set a disc provider if web3manager doesnt exist, do we maybe wanna log or raise something here if that happens?